runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -32,6 +32,11 @@ pub mod state {
     pub const CODES: &[u8] = &[0x02];
     /// Prefix for Ethereum account storage in our storage (maps H160||H256 -> H256).
     pub const STORAGES: &[u8] = &[0x03];
+
+    /// Prefix for Ethereum block hashes(only for last 256 blocks excluding current) 
+    /// storage in our storage (maps H160||U256 -> H256).
+    pub const HASHES: &[u8] = &[0xe7];
+    pub const BLOCK_TH: u64 = 256;
```

Comment:
Maybe call this something like `BLOCK_HASH_WINDOW_SIZE` and add a documentation comment.